lib/auth/clt.go

```diff
@@ -2105,6 +2105,16 @@ func (c *Client) UpsertAppSession(ctx context.Context, session services.WebSessi
 	return trace.NotImplemented(notImplementedMessage)
 }
 
+// ResumeAuditStream resumes existing audit stream.
+func (c *Client) ResumeAuditStream(ctx context.Context, sid session.ID, uploadID string) (events.Stream, error) {
+	return c.APIClient.ResumeAuditStream(ctx, string(sid), uploadID)
+}
+
+// CreateAuditStream creates new audit stream.
+func (c *Client) CreateAuditStream(ctx context.Context, sid session.ID) (events.Stream, error) {
+	return c.APIClient.CreateAuditStream(ctx, string(sid))
+}
+
```

Comment:
Can you mark these two as deprecated and to remove in 7.0.0? Then we can remove these wrappers from `lib/auth/clt.go`.

Reply:
I added this to #5251 since this is already merged.